api/envoy/config/tap/v3/common.proto

```diff
@@ -247,6 +249,16 @@ message OutputSink {
     // GrpcService to stream data to. The format argument must be PROTO_BINARY.
     // [#comment: TODO(samflattery): remove cleanup in uber_per_filter.cc once implemented]
     StreamingGrpcSink streaming_grpc = 4;
+
+    // Tap output will be buffered in a single block before flushing to the :http:post:`/tap` admin endpoint
+    //
+    // .. attention::
+    //
+    //   It is only allowed to specify the buffered admin output sink if the tap is being
+    //   configured from the :http:post:`/tap` admin endpoint. Thus, if an extension has
+    //   been configured to receive tap configuration from some other source (e.g., static
+    //   file, XDS, etc.) configuring the buffered admin output type will fail.
+    BufferedAdminSink buffered_admin = 5;
```

Comment:
Please add a release note in current.rst with info about this new feature, and ref link to this field.